static/styles/portico/portico.css

```diff
@@ -1,3 +1,10 @@
+// $blue2: #3a87ad;
```

Comment:
**To test this PR with different variations**
Just uncomment the two and comment the two pairs that you want to try.